posthog/batch_exports/http.py

```diff
@@ -127,6 +128,26 @@ def safely_get_queryset(self, queryset):
         queryset = queryset.filter(batch_export_id=self.kwargs["parent_lookup_batch_export_id"])
         return queryset
 
+    @action(methods=["POST"], detail=True, required_scopes=["batch_export:write"])
+    def retry(self, *args, **kwargs) -> response.Response:
```

Comment:
The big difference with using this action is that we don't have to worry about converting the datetime bounds: They are already in the schedule's timezone, so we just pass them along.